bin/vitamin.js

```diff
@@ -147,8 +146,8 @@ const build = (options, hotCallback, restartServer) => (options.hot ?
 
 
 const test = (args) => {
-    npmRunSpawn(
-        'jest',
+    spawn(
+        vitaminResolve('node_modules', '.bin', 'jest'),
```

Comment:
when you resolve it with `npmRun` what is the sequence with local, global and dependencies?